### ISSUE TITLE
Repeller visual issues

### ISSUE THREAD

**Issue by Uub112** [TheCyberBrick/The-Betweenlands#1050]
#### Betweenlands version:
3.6.0

#### Forge version:
14.23.5.2854

#### Singleplayer or Multiplayer:
Singleplayer

#### Installed mods:
The Betweenlands

Two issues here:
    1. The repeller's shield doesn't seem to render, only the rune particles that are projected from it.
    2. When you go into either F5 third-person perspective (or the F1 gui-hiding perspective) while a repeller is active and rendered, your perspective turns to a blank white.

#### Pictures:
![2020-05-12_17 35 16](https://user-images.githubusercontent.com/48371065/81759146-bd3b7c00-9478-11ea-954b-c4a4743c716a.png)
The runes without the shield

![2020-05-12_17 46 43](https://user-images.githubusercontent.com/48371065/81759150-bf9dd600-9478-11ea-8ee9-93f3d8e9c6c4.png)
Third person facing repeller, out of render range
![2020-05-12_17 47 39](https://user-images.githubusercontent.com/48371065/81759151-bf9dd600-9478-11ea-8fd5-74c093e49924.png)
First person facing repeller, just in render range
![2020-05-12_17 47 42](https://user-images.githubusercontent.com/48371065/81759152-c0366c80-9478-11ea-8bdb-bb8933372119.png)
Third person facing repeller, just in render range
![2020-05-12_17 47 46](https://user-images.githubusercontent.com/48371065/81759153-c0366c80-9478-11ea-8271-586237d80d43.png)
Third person facing selfward, just in render range

**Comment by theishiopian** [comment]
That GPU might not support the shaders used, just a guess.

**Comment by TheCyberBrick** [comment]
Yes, it's likely that this is caused by an incompatibility with your Intel graphics card. Could you send us your client log?

**Comment by DoctorCJack** [comment]
> Yes, it's likely that this is caused by an incompatibility with your Intel graphics card. Could you send us your client log?

I am experiencing the same bug but in version 3.7.3, I can send you my client log.
https://www.mediafire.com/file/l3ykx8houqv48a0/latest.log/file
